toolkit/tools/pkgworker/pkgworker.go

```diff
@@ -54,6 +54,7 @@ var (
 	packagesToInstall    = app.Flag("install-package", "Filepaths to RPM packages that should be installed before building.").Strings()
 	outArch              = app.Flag("out-arch", "Architecture of resulting package").String()
 	useCcache            = app.Flag("use-ccache", "Automatically install and use ccache during package builds").Bool()
+	maxCPU               = app.Flag("max-cpu", "Max number of CPUs used for package building").String()
```

Comment:
Can we either mark this as required or provide a default value of `0`, if not provided?

Reply:
That's a good idea; I'll fix up the default case. will check for when nothing is provided, and not set the macro at all in that case.
Should not need to mark `max-cpu` as required, since it's just an override